main/src/ca/uwaterloo/flix/api/lsp/provider/CompletionProvider.scala

```diff
@@ -292,7 +292,8 @@ object CompletionProvider {
       case ResolutionError.UndefinedName(_, _, _, isUse, _) => if (isUse) (1, SyntacticContext.Use) else (2, SyntacticContext.Expr.OtherExpr)
       case ResolutionError.UndefinedType(_, _, _) => (1, SyntacticContext.Type.OtherType)
       case WeederError.MalformedIdentifier(_, _) => (2, SyntacticContext.Import)
-      case ParseError(_, ctx, _) => (5, ctx)
+      case UnexpectedToken(_, _, ctx, _, _) => (5, ctx)
+      case ParseError(_, ctx, _, _) => (5, ctx)
```

Comment:
Whats the reason for these two additions? How do they impact auto completion?

Reply:
Before we had only `ParseError`, but I split out the most common use-case into the new `UnexpectedToken`.
Adding `UnexpectedToken` with the same priority(5) as `ParseError` just replicates the behavior we had before.

Reply:
I see, I'll do another revision then.